docs/sites.yml

```diff
@@ -3154,3 +3154,16 @@
   built_by: Gabe Ragland
   built_by_url: https://twitter.com/gabe_ragland
   featured: false
+- title: Disrupting Nate
+  description: >
+    Ketogenic Diet, Podcasts, and Blockchain.
+  url: https://www.disruptingnate.com/
```

Comment:
This appears to be down for me?

Reply:
Was deploying and still working out a DNS issue, need to type in www